curriculum/challenges/english/25-front-end-development/lecture-working-with-audio-and-video/6733d83630e76f08ff49e6dc.md

```diff
@@ -2,31 +2,35 @@
 id: 6733d83630e76f08ff49e6dc
 title: What Is the HTMLMediaElement API and How Does It Work?
 challengeType: 19
-# videoId: nVAaxZ34khk
+videoId: NS3YH1_BUeo
 dashedName: what-is-the-htmlmediaelement-api-and-how-does-it-work
 ---
 
 # --description--
 
-The video for this lecture isn't available yet, one will be available soon. Here is a transcript of the lecture for now:
+Watch the video and answer the questions below.
 
-The HTMLMediaElement API is a powerful tool to control the behavior of audio and video elements on your page. It extends the base HTMLElement interface, so you have access to the base properties as well as these helpful methods.
+# --transcript--
 
-You've already explored some of the methods in a previous lecture about the Audio element, such as `play()` and `pause()`. These same methods are available on video elements as well.
+What Is the HTMLMediaElement API and How Does It Work?
+
+The `HTMLMediaElement API` is a powerful tool to control the behavior of audio and video elements on your page. It extends the base `HTMLElement` interface, so you have access to the base properties as well as these helpful methods.
```

Comment:
```suggestion
What is the `HTMLMediaElement` API and how does it work?

The `HTMLMediaElement` API is a powerful tool to control the behavior of audio and video elements on your page. It extends the base `HTMLElement` interface, so you have access to the base properties as well as these helpful methods.
```
I would not include API in the backticks as it can go before HTMLMediaElement depending on the language